lib/ansible/modules/database/mysql/mysql_replication.py

```diff
@@ -103,6 +103,16 @@
         required: false
         default: null
         version_added: "2.0"
+    master_use_gtid:
+        description:
+            - MariaDB Global Transaction ID parameter
+        required: False
+        choices:
+            - current_pos
+            - slave_pos
+            - no
+        default: null
+        version_added: "2.4"
```

Comment:
`version_added` should be 2.7 for now
@bobuss If you can fix this quickly, I'll review ASAP.